regenerate_api.py

```diff
@@ -88,7 +88,7 @@ def check_openapi_version():
     """
     Check compatible OpenAPI version
     """
-    command = [*OPENAPI_COMMAND, '--version']
+    command = [*OPENAPI_COMMAND, 'version']
```

Comment:
This will cause issues in the deploy because it's using an older version with `--version` - can we support both using a loop or update the deploy instructions.

Reply:
Instructions updated, both supported.